source/server/http/admin.cc

```diff
@@ -275,6 +276,24 @@ Http::Code AdminImpl::handlerClusters(const std::string&, Http::HeaderMap&,
   return Http::Code::OK;
 }
 
+// TODO(jsedgwick) Use query params to list available dumps, selectively dump, etc.
```

Comment:
Sorry I didn't notice this till just now, but per the docs this should only respond to a POST. Can we fix that? Since we are probably going to have more POST methods in the future (cc @jmarantz) it might be worth it to build this into the registration system somehow, but up to you.

Reply:
Oops sorry I was crossing streams with https://github.com/envoyproxy/envoy/pull/2837. This comment applies there. :)

Reply:
ok following up there.